Assets/AltTester/AltDriver/Common/AltObject.cs

```diff
@@ -47,17 +47,28 @@ public AltObject GetParent()
             CommHandler.SleepFor(CommHandler.GetDelayAfterCommand());
             return altObject;
         }
-
+        public AltObject getParent()
```

Comment:
Mark the methods Obsolete so users will know to use the other method. Here is how you can do it: https://stackoverflow.com/questions/1759352/how-to-mark-a-method-as-obsolete-or-deprecated